src/plugin.ts

```diff
@@ -0,0 +1,199 @@
+import { SchemaConfig, SchemaBuilder, BuilderLens } from "./builder";
+import {
+  ASTKindToNode,
+  GraphQLResolveInfo,
+  GraphQLFieldConfig,
+  GraphQLFieldResolver,
+  GraphQLObjectTypeConfig,
+  GraphQLInterfaceTypeConfig,
+  Visitor,
+} from "graphql";
+import { withNexusSymbol, NexusTypes, Omit } from "./definitions/_types";
+import { isPromiseLike, PrintedTypeGen } from "./utils";
+import {
+  NexusFieldExtension,
+  NexusObjectTypeExtension,
+  NexusSchemaExtension,
+  NexusInterfaceTypeExtension,
+} from "./extensions";
+import { NexusTypeExtensions } from "./definitions/decorateType";
+
+export type CreateFieldResolverInfo = {
+  /**
+   * The internal Nexus "builder" object
+   */
+  builder: SchemaBuilder;
+  /**
+   * Info about the GraphQL Field we're decorating.
+   * Always guaranteed to exist, even for non-Nexus GraphQL types
+   */
+  fieldConfig: Omit<GraphQLFieldConfig<any, any>, "resolve"> & {
+    extensions: { nexus: NexusFieldExtension };
+  };
+  /**
+   * The config provided to the Nexus type containing the field.
+   * Will not exist if this is a non-Nexus GraphQL type.
+   */
+  parentTypeConfig:
+    | Omit<GraphQLObjectTypeConfig<any, any>, "fields"> & {
+        extensions: { nexus: NexusObjectTypeExtension };
+      }
+    | Omit<GraphQLInterfaceTypeConfig<any, any>, "fields"> & {
+        extensions: { nexus: NexusInterfaceTypeExtension };
+      };
+  /**
+   * The root-level SchemaConfig passed
+   */
+  schemaConfig: Omit<SchemaConfig, "types"> & {
```

Comment:
Since #230 you can now use `InternalSchemaConfig` and be rid of the `Omit ... "types"` stuff.